### PR TITLE
EC-0000 Update to Rebus 7

### PR DESCRIPTION
**Pull request by danielmallott**
And the rest of the dependencies

### CODE REVIEW

**Comment by danielmallott** [comment]
NOTE: This will be tagged as v3.0.0 since Rebus 7 contains breaking changes.

**Comment by danielmallott** [comment]
This will be for Outbox/microservices. Monolith is still blocked by Autofac 🙈

File: src/Rebus.Datadog.Tracing/Rebus.Datadog.Tracing.csproj [hunk 1]
```diff
@@ -11,8 +11,8 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Datadog.Trace" Version="2.5.1" />
-        <PackageReference Include="Rebus" Version="6.6.2" />
+        <PackageReference Include="Datadog.Trace" Version="2.24.1" />
```
Comment (mihaitabamburic) [inline src/Rebus.Datadog.Tracing/Rebus.Datadog.Tracing.csproj:14]: This probably shouldn't be upgraded unless there are breaking changes

Comment (danielmallott) [reply src/Rebus.Datadog.Tracing/Rebus.Datadog.Tracing.csproj:14]: But why leave it on an old version?

Comment (danielmallott) [reply src/Rebus.Datadog.Tracing/Rebus.Datadog.Tracing.csproj:14]: Mainly because ideally, it matches the version installed in the consuming application

Comment (mihaitabamburic) [reply src/Rebus.Datadog.Tracing/Rebus.Datadog.Tracing.csproj:14]: Let's go with that then 😄

**Review by mihaitabamburic — APPROVED** [review]

**Pull request merged**